### PR TITLE
Alpha

### PR DESCRIPTION
**Pull request by LaGaia**

### CODE REVIEW

File: alpha.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#ALPHA
+
+The 'Alpha' phase of the consent-form builder has been the first example of handing a project over between the service design and product teams.
+
+This has involved changes to working practices, as well as changes to the overall product. 
+
+## Changes to working practices
+
+### Turning user stories into Cucumber
+<<Niklas: two words on this?>>
```
Comment (sl33p) [inline alpha.md:10]: It's Nicklas. :P

Comment (LaGaia) [reply alpha.md:10]: :P any other words? :)

**Comment by sl33p** [comment]
I'm gonna merge this and then come back and add content where you need it. ✨

**Comment by LaGaia** [comment]
EXCELLENT

**Pull request merged**